homeassistant/components/nuki/binary_sensor.py

```diff
@@ -0,0 +1,82 @@
+"""Doorsensor Support for the Nuki Lock."""
+
+import logging
+
+from pynuki import STATE_DOORSENSOR_OPENED
+
+from homeassistant.components.binary_sensor import DEVICE_CLASS_DOOR, BinarySensorEntity
+
+from . import NukiEntity
+from .const import (
+    ATTR_LOCK_DOORSENSOR_STATE,
+    ATTR_LOCK_DOORSENSOR_STATE_NAME,
+    ATTR_NUKI_ID,
+    DATA_COORDINATOR,
+    DATA_LOCKS,
+    DOMAIN,
```

Comment:
DOMAIN in platforms are exclusive to the platform itself (binary_sensor)
```suggestion
    DOMAIN as NUKI_DOMAIN,
```